code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -113,18 +125,64 @@
-						/datum/reagent/mercury = 1,
-						/datum/reagent/medicine/dexalin = 8,
-						/datum/reagent/medicine/inaprovaline = 8,
-						/datum/reagent/toxin = 4)
+	list_reagents = list(/datum/reagent/medicine/hyperzine = 4, 
+							/datum/reagent/medicine/dexalin = 8, 
+							/datum/reagent/medicine/inaprovaline = 8, 
+							/datum/reagent/toxin = 5)
 
 /obj/item/reagent_containers/hypospray/autoinjector/hyperzine
 	name = "hyperzine autoinjector"
 	desc = "An auto-injector freshly loaded with a safe-to-use hyperzine mix."
 	amount_per_transfer_from_this = 25
 	volume = 25
 	icon_state = "autoinjector-1" //TEMP, god willing
-	list_reagents = list(/datum/reagent/medicine/hyperzine = 5,
-						/datum/reagent/medicine/dexalin = 10,
-						/datum/reagent/medicine/inaprovaline = 10)
+	list_reagents = list(/datum/reagent/medicine/hyperzine = 5, 
+							/datum/reagent/medicine/dexalin = 10, 
+							/datum/reagent/medicine/inaprovaline = 10)
+
+/obj/item/reagent_containers/hypospray/autoinjector/synaptizine
+	name = "synaptizine autoinjector"
+	desc = "An auto-injector loaded with a safe-to-use synaptizine mix, a good anti-stun chemical."
+	volume = 20
+	icon_state = "autoinjector-10"
+	list_reagents = list(/datum/reagent/medicine/synaptizine = 5, 
+							/datum/reagent/medicine/tricordrazine = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/synaptizine/expired
+	name = "expired synaptizine autoinjector"
+	desc = "An auto-injector loaded with a safe-to-use synaptizine mix, 3 months past it's expiration date."
+	icon_state = "autoinjector-10"
+	list_reagents = list(/datum/reagent/medicine/synaptizine = 3, 
+							/datum/reagent/medicine/tricordrazine = 11, 
+							/datum/reagent/toxin = 6)
+
+/obj/item/reagent_containers/hypospray/autoinjector/neuraline
+	name = "neuraline autoinjector"
+	desc = "An auto-injector loaded with 3 uses of Neuraline, a very potent short-lived anti-stun chemical. Overdosing will cause brain damage."
+	amount_per_transfer_from_this = 3
+	volume = 9
+	icon_state = "autoinjector-6"
+	list_reagents = list(/datum/reagent/medicine/neuraline = 9)
+
+/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus
+	name = "peridaxon plus autoinjector"
+	desc = "An auto-injector loaded with 3 seperate injections of Peridaxon Plus, a miracle drug for healing internal damage. !!DO NOT INJECT MORE THAN ONCE AT A TIME!!"
+	amount_per_transfer_from_this = 1
+	volume = 3
+	icon_state = "autoinjector-1"
+	list_reagents = list(/datum/reagent/medicine/peridaxon_plus = 3)
+
+/obj/item/reagent_containers/hypospray/autoinjector/quickclot_plus
+	name = "quickclot plus autoinjector"
+	desc = "An auto-injector loaded with 3 seperate injections of Quickclot Plus, a miracle drug for healing internal bleeding. !!DO NOT INJECT MORE THAN ONCE AT A TIME!!"
+	amount_per_transfer_from_this = 1
+	volume = 3
+	icon_state = "autoinjector-7"
+	list_reagents = list(/datum/reagent/medicine/quickclot_plus = 3)
+
+
+
+
+
+
```

Comment:
Could you avoid this bunch of whitespaces?

Reply:
aaaaa

I thought I removed these








aaaaa








ok

Reply:
done